ppr-ui/src/components/registration/RegistrationLengthTrust.vue

```diff
@@ -155,7 +266,37 @@ export default defineComponent({
       showErrorComponent: computed((): boolean => {
         return (lengthTrust.showInvalid)
       }),
+      showErrorLienAmount: computed((): boolean => {
+        return (lengthTrust.showInvalid && lengthTrust.lienAmount === '')
+      }),
+      showErrorSurrenderDate: computed((): boolean => {
+        return (lengthTrust.showInvalid && lengthTrust.surrenderDate === '')
+      }),
+      lienAmountMessage: computed((): string => {
+        if (lengthTrust.showInvalid && lengthTrust.lienAmount === '') {
+          return 'This field is required'
+        }
+        return ''
+      }),
+      surrenderDateMessage: computed((): string => {
+        if (lengthTrust.showInvalid && lengthTrust.surrenderDate === '') {
+          return 'This field is required'
+        }
+        return ''
+      }),
+      minSurrenderDate: computed((): string => {
+        var dateOffset = (24 * 60 * 60 * 1000) * 21
```

Comment:
perhaps a comment here on what this is :)

Reply:
Comment added